### ISSUE TITLE
Fetch the mobile number from users profile

### ISSUE THREAD

**Issue by morph027** [nextcloud/twofactor_gateway#98]
If the user already has a mobile number configured in it's personal settings, you might want to fetch it from there.

**Comment by ChristophWurst** [comment]
Absolutely. We should prefill it for Signal and SMS. The only problem I see is that some gateways require a specific format we should add a hint for.
